<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Animation Path :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="animAssetPath_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Character Path :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="characterAssetPath_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Skeletal Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skeletalPath_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="importCharacter_pushButton">
       <property name="text">
        <string>Import Character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importAnim_pushButton">
       <property name="text">
        <string>Import Anim</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
